elastic/kibana
# [Inventory] Change discover link to use entity definition

Closes #200595 

## Summary

This PR changes the link to discover: 
- Pass a different data view (using the entity definition indices)
- from Open in Discover to Explore in Discover

## Testing

- To have the test data I used metricbeat (for the host entity) and synthtrace (for the other entities type)
- Open the inventory page and expand an entity type
- Click on the `...` in the actions column
- Click on `Explore in Discover`

The results should be filtered by the selected entity and the dataview should include the indices from the entity definition:

https://github.com/user-attachments/assets/29bb46b4-719e-4874-b1aa-056ac28d234a


https://github.com/user-attachments/assets/36c08b8b-f507-445e-a7b5-8eb1176beb90


With service type filter (should not persist)

https://github.com/user-attachments/assets/ae6ef28a-e55a-4a32-9128-802ebea25607
